source/GameData.cpp

```diff
@@ -946,6 +958,19 @@ void GameData::LoadFile(const string &path, bool debugMode)
 			for(const DataNode &child : node)
 				list.push_back(child.Token(0));
 		}
+		else if(key == "categories" && node.Size() >= 2)
+		{
+			if(node.Token(1) == "outfit" || node.Token(1) == "ship")
```

Comment:
Bind a reference to the correct list, i.e.
`container<string> &categoryList = node.Token(1) == "outfit" ? outfitCategories : shipCategories;`

and then you don't need to duplicate your push_back call, and can still perform uniqueness checks on the list (if using vector, `find(token) == end()`, or something like unsorted_set, `! .count(token)`